.azure-pipelines/integration-tests.yml

```diff
@@ -145,15 +143,12 @@ jobs:
   strategy:
     matrix:
       netcoreapp2_1:
-        dotnetCoreSdkVersion: 2.1.x
```

Comment:
Note to reviewers: SDKs not needed here, they are installed into the Linux containers.